lib/components/form/user-settings.js

```diff
@@ -38,14 +38,35 @@ function formatElapsedTime(timestamp, intl) {
   const hours = allHours % 24
   const days = (allHours - hours) / 24
 
-  // This text will be shown in a tooltip, therefore
+  // This text can be shown in a tooltip as well as within HTML tags, therefore
   // it is obtained using formatMessage rather than <FormattedfMessage>.
-  return intl
-    .formatMessage(
-      { id: 'common.time.fromNowUpdate' },
-      { days, hours, minutes }
+  let duration
+  if (days === 0) {
+    if (hours === 0) {
+      if (minutes === 0) {
```

Comment:
Real nitpick: couldn't we avoid all the else blocks if we put the conditions currently in the else blocks above the statements currently right after the if blocks?

Reply:
Good suggestion, it's much cleaner that way (https://github.com/opentripplanner/otp-react-redux/pull/851/commits/2eb8d65740d2e3bf346c914811392329d87db26a).